project/MimaSettings.scala

```diff
@@ -25,6 +25,14 @@ object MimaSettings {
   )
 
   private def otherProblems = Seq(
-  )
+    "org.scalacheck.Shrink.shrinkTuple2",
+    "org.scalacheck.Shrink.shrinkTuple3",
+    "org.scalacheck.Shrink.shrinkTuple4",
+    "org.scalacheck.Shrink.shrinkTuple5",
+    "org.scalacheck.Shrink.shrinkTuple6",
+    "org.scalacheck.Shrink.shrinkTuple7",
+    "org.scalacheck.Shrink.shrinkTuple8",
+    "org.scalacheck.Shrink.shrinkTuple9"
```

Comment:
What makes the new generated methods incompatible with the current ones, could you clarify please? At first glance, the generated method should be very similar to those that they are supposed to replace.